armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -860,6 +860,7 @@ def determineTargetComponent(self, b, flagOfInterest=None):
                 f"Block {b}\nflagOfInterest {flagOfInterest}\nComponents {componentWFlag}"
             )
         self._componentDeterminesBlockHeight[componentWFlag[0]] = True
+        b.p.axialExpTargetComponent = componentWFlag[0]
```

Comment:
Note to self, `b.p.axialExpTargetComponent` might make `self._componentDeterminesBlockHeight` obsolete.